core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -95,10 +101,27 @@ trait RawlsInstrumented extends DefaultInstrumented {
     }
   }
 
+  // Handy counter definitions which can be used by implementing classes:
+
+  protected def submissionStatusCounterProvider(workspaceName: WorkspaceName) =
+    (status: SubmissionStatus) => ExpandedMetricBuilder
+      .expand(WorkspaceMetric, workspaceName)
+      .expand(SubmissionStatusMetric, status)
+      .asCounter()
+
+  protected def workflowStatusCounterProvider(workspaceName: WorkspaceName, submissionId: UUID) =
```

Comment:
but this is `SubmissionMonitorActor.workflowStatusCounter` by another name, no? Can't we move that implicit somewhere shared and reuse it?